fdbserver/CoordinatedState.h

```diff
@@ -23,44 +23,12 @@
 #pragma once
 
 #include "fdbclient/FDBTypes.h"
-
-class CoordinatedState : NonCopyable {
-public:
-	// Callers must ensure that any outstanding operations have been cancelled before destructing *this!
-	CoordinatedState( class ServerCoordinators const& );
-	~CoordinatedState();
-
-	Future<Value> read();
-	// May only be called once.
```

Comment:
These API contracts were lost. Can you find a place for them?

Reply:
And if you can somehow keep them in the header, code intelliSense might be able to pick up the docs and show them nicely at the call sites.

Reply:
I added back these comments. This class is never used outside `CoordinatedState.actor.cpp`, which is why I removed it from the header file.